ios/DomainLayer/SharedDomain/Package.swift

```diff
@@ -52,6 +55,10 @@ let package = Package(
                 "SharedDomainMocks"
             ]
         ),
+//        .plugin(
+//            name: "KMPBuildPlugin",
+//            capability: .buildTool()
+//        ),
```

Comment:
Are we keeping the commented out code on purpose for the future?

Reply:
Yeah.. but it's true that we shouldn't keep non-functional code in the main branch 👍 
I've deleted the plugin and moved the code to [`ios/feature/build-kmp-plugin`](https://github.com/MateeDevs/devstack-native-app/tree/ios/feature/build-kmp-plugin) branch